extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/devconsole/StaticResourceInfo.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.resteasy.reactive.server.deployment.devconsole;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class StaticResourceInfo {
+
+    public Map<String, StaticFile> resourceMap = new TreeMap<>();
+
+    public static final class StaticFile {
+        public String lastName;
+        public boolean isFolder;
+        public List<StaticFile> children = new ArrayList<>();
```

Comment:
It would be nice to have the children sorted, can you make them `Comparable` and sort them on `postVisit`?

Traditionally the sort order is:

- Folders first
- Then sort alphabetically

Reply:
Done :)